subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -582,6 +582,7 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 		{"key2-pwd", required_argument, 0, 'K'},
 		{"suiteb-type", required_argument, 0, 'S'},
 		{"TLS-cipher", required_argument, 0, 'T'},
+		{"use-ca", required_argument, 0, 'A'},
```

Comment:
why not use `verify-peer-cert`? It's gives more context than just using CA.

Reply:
Updated, thanks